Info/Scripts/Settings.cs

```diff
@@ -5,729 +5,734 @@ public static class MySettings
-	// of their home like trees, shrubs, fences, lave, water, and other items. Lawn tools require an amount of
-	// gold to place items. If this was previously true and characters placed lawn items, and then you set it to
-	// false, the lawn items will refund the gold back to the character's bank box and the lawn tools will be
-	// removed from the game.
+		// When true (default setting), characters can use lawn tools (from architects) to add items to the outside
+		// of their home like trees, shrubs, fences, lave, water, and other items. Lawn tools require an amount of
+		// gold to place items. If this was previously true and characters placed lawn items, and then you set it to
+		// false, the lawn items will refund the gold back to the character's bank box and the lawn tools will be
+		// removed from the game.
 
-		public static bool S_LawnsAllowed = false;
+		public static bool S_LawnsAllowed = true;
 
-	// When true (default setting), characters can use remodeling tools (from architects) to add items to their
-	// home like walls, doors, tiles, and other items. Remodeling tools require an amount of gold to place items.
-	// If this was previously true and characters placed remodeling items, and then you set it to false, the
-	// remodeling items will refund the gold back to the character's bank box and the remodeling tools will be
-	// removed from the game.
+		// When true (default setting), characters can use remodeling tools (from architects) to add items to their
+		// home like walls, doors, tiles, and other items. Remodeling tools require an amount of gold to place items.
+		// If this was previously true and characters placed remodeling items, and then you set it to false, the
+		// remodeling items will refund the gold back to the character's bank box and the remodeling tools will be
+		// removed from the game.
 
-		public static bool S_ShantysAllowed = false;
+		public static bool S_ShantysAllowed = true;
 
-	// The number of days, no less than 5.0 (decimal format), that a boat or magic carpet will decay if on
-	// the sea not used.
+		// The number of days, no less than 5.0 (decimal format), that a boat or magic carpet will decay if on
+		// the sea not used.
 
 		public static double S_BoatDecay = 365.0;
 
-	// The number of days, no less than 30.0 (decimal format), that a home will decay if an owner never shows up.
+		// The number of days, no less than 30.0 (decimal format), that a home will decay if an owner never shows up.
 
 		public static double S_HomeDecay = 365.0;
 
-	// If false, then houses never decay and the above setting is ignored.
+		// If false, then houses never decay and the above setting is ignored.
 
 		public static bool S_HousesDecay = false;
 
-	// The amount of houses an account's characters may own. A -1 setting will be unlimited.
+		// The amount of houses an account's characters may own. A -1 setting will be unlimited.
 
 		public static int S_HousesPerAccount = 1;
```

Comment:
public static int S_HousesPerAccount = 5;

Reply:
Still you'll have 1 empty xD